rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date, :books, :person
+
+  def initialize(date, books, person)
+    @date = date
+    @books = []
```

Comment:
- [ ] A rental belongs to a book, please 🙏🏽 correct this line of the code `@book = book` to create the correct relationship between rental and book.